coordinator/manifest/manifest.go

```diff
@@ -496,6 +499,19 @@ func (m Manifest) Check(zaplogger *zap.Logger) error {
 		}
 	}
 
+	var manifestUpdaters int
+	for _, mrUser := range m.Users {
+		for _, roleName := range mrUser.Roles {
+			if m.Roles[roleName].ResourceType == "Manifest" && strings.ToLower(m.Roles[roleName].Actions[0]) == user.PermissionUpdateManifest {
+				manifestUpdaters++
+				break // Avoid counting the same user multiple times if they are assigned more than one role with update permission
+			}
+		}
+	}
+	if manifestUpdaters < int(m.Config.UpdateThreshold) {
```

Comment:
```suggestion
	var manifestUpdaters uint
	for _, mrUser := range m.Users {
		for _, roleName := range mrUser.Roles {
			if m.Roles[roleName].ResourceType == "Manifest" && strings.ToLower(m.Roles[roleName].Actions[0]) == user.PermissionUpdateManifest {
				manifestUpdaters++
				break // Avoid counting the same user multiple times if they are assigned more than one role with update permission
			}
		}
	}
	if manifestUpdaters < m.Config.UpdateThreshold {
```
nit: change type to avoid cast